src/components/InfiniteHits.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div :class="suit()" v-if="state">
+    <slot :items="items" :isLastPage="isLastPage" :refine="refine">
+      <ul :class="suit('list')">
```

Comment:
not related to this change, but I'm having second thoughts on putting results in ul/li. Gov.uk for example uses div > article, and I think that makes more sense. cc @ronanlevesque

Reply:
I've also seen ol > li, which also makes more sense since results are ordered

Reply:
Yep why not, `ol > li` makes probably more sense because like you said results are ordered.